### PR TITLE
Add plugins

### PR DESCRIPTION
**Pull request by elfiesmelfie**
Add a new batch of plugins.

### CODE REVIEW

File: templates/amqp1.conf.j2 [hunk 1]
```diff
@@ -1,14 +1,22 @@
 #TODO: Add new value for Host and port
 #TODO: create a loop to parse input and generate the Instance members
 
+{% if plugin_amqp1_interval is not defined %}
 LoadPlugin amqp1
+{% else %}
+<LoadPlugin amqp1>
+   Interval {{ collectd_plugin_amqp1_interval }}
```
Comment (mrunge) [inline templates/amqp1.conf.j2:8]: Interval belongs to the plugin section?

File: templates/connectivity.conf.j2 [hunk 2]
```diff
@@ -1,10 +1,18 @@
+{% if collectd_plugin_connectivity_interval is not defined %}
+LoadPlugin connectivity
+{% else %}
+<LoadPlugin connectivity>
+   Interval {{ collectd_plugin_connectivity_interval }}
```
Comment (mrunge) [inline templates/connectivity.conf.j2:5]: same? I'd put the interval to the plugin section itself.

**Review by mrunge — DISMISSED** [review]
Thank you for your work here.

My recommendation would be to get this merged asap and to fix issues when they are being identified.

File: templates/amqp1.conf.j2 (hunk 1 above)
Comment (elfiesmelfie) [reply templates/amqp1.conf.j2:8]: That's what I thought initially, however, all the sample configs show Interval in the LoadPlugin section.

It's documented under the `Global Options` section and is an option for `LoadPlugin`: https://collectd.org/documentation/manpages/collectd.conf.5.shtml#global_options

Comment (mrunge) [reply templates/amqp1.conf.j2:8]: In my understanding, you can have the interval in the global section and also for every plugin.

**Review by mrunge — APPROVED** [review]
thank you

**Pull request merged**